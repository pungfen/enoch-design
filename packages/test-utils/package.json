{
  "name": "@enochfe/test-utils",
  "author": "pung <fp94214@gmail.com>",
  "description": "enoch test-utils",
  "version": "1.0.1",
  "type": "module",
  "main": "src/index.ts",
  "scripts": {
    "build": "tsup"
  },
  "dependencies": {
    "@vitejs/plugin-vue-jsx": "^2.0.1",
    "esbuild": "^0.15.10",
    "rollup": "^3.1.0",
    "rollup-plugin-esbuild": "^4.10.1",
    "unplugin": "^0.9.6",
    "unplugin-vue": "3.1.3"
  },
  "devDependencies": {
    "vite": "^3.1.3",
    "vue": "^3.2.39"
  }
}
