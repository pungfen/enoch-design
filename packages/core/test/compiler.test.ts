import { mount } from '@vue/test-utils'
import Demo from './demo.vue'

test('mount', async () => {
  expect(Demo).toBeTruthy()

  const wrapper = mount(Demo)

  expect(wrapper.html()).toMatchSnapshot()
})
