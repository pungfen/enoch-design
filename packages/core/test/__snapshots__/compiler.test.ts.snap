// Vitest Snapshot v1

exports[`mount 1`] = `"<div>{}</div>"`;
